org-cyf/content/guides/employability/portfolio/portfolio-criteria.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Portfolio criteria
```

Comment:
I think all this stuff should go into the Portfolio module now

Reply:
I'm happy to port these critiera to the other module but need to check in with portfolio peeps to determine what order they want to do these things in on the portfolio module (if it matters at all)